CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+              "type": "string",
+              "description": "Script on-chain address"
+            }
+            "required": [
+                "fullScriptHash",
+                "scriptHash",
+                "contractAddress"
+              ]
+        }
+      ]
+    }
+  },
+
+  "required": [
+    "subject",
+    "schemaVersion",
+    "certificationLevel",
+    "certificateIssuer",
+    "report",
+    "summary",
+    "disclaimer",
+    "scripts"
+  ]
+}
+```
+
+### Example
+
+```json
+{
+  "subject": "d684512ccb313191dd08563fd8d737312f7f104a70d9c72018f6b0621ea738c5b8213c8365b980f2d8c48d5fbb2ec3ce642725a20351dbff9861ce9695ac5db8",
+  "schemaVersion": "1.0.0",
+  "certificationLevel": 1,
+  "certificateIssuer": {
+    "name": "Audit House LLC",
+    "logo": "https://www.example.com/media/logo.svg"
+    "social": {
+        "contact": "contact@example.com",
+        "link": "https://example.com",
+        "twitter": "twiterHandle",
+        "github": "githubHandle",
+        "website": "https://www.example.com"
+    }
+  },
+  "report": {
+    "reportURLs": [
+        "https://example.io/certificate.pdf",
+        "ipfs://bafybeiemxfal3jsgpdr4cjr3oz3evfyavhwq"
+        ],
+    "reportHash": "c6bb42780a9c57a54220c856c1e947539bd15eeddfcbf3c0ddd6230e53db5fdd"
+  },
+  "summary": "This is the summary of the report."
+  "disclaimer": "This is the legal disclaimer from the report",
+  "scripts": [
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "711dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "scriptHash": "1dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "contractAddress": "addr1wywukn5q6lxsa5uymffh2esuk8s8fel7a0tna63rdntgrysv0f3ms"
+        },
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "384da5375661cb1e07713eea236cd681921dcb4e80d7cd0ed4e7feebd7",
+          "scriptHash": "6cd68191dcb4e80d7c5661cb1e074e7feebd73eea232d0ed384da537",
+          "contractAddress": "addr1wywukn5q6lrdntgrysv0f7a0tna63ymffh23ms5uxsaesuk8s8fel"
+        }
+  ]
+}
+```
+
+The metadata should be discoverable by all certification stakeholders, including end-users, DApp developers, and ecosystem components, such as light wallets and DApp stores. Information should be indexable by certification issuer, DApp developer, DApp and DApp version.
```

Comment:
> The metadata should be discoverable

How is the off-chain metadata meant to be stored and served to these users?

Reply:
@RSoulatIOHK @Ryun1 - I still can't find an elaboration of the current sentence here: https://github.com/RSoulatIOHK/CIPs/blob/cip-certification-metadata/CIP-0096/README.md#off-chain-metadata-format
> **Off-chain Metadata Format**
> The Dapp Certification certificate is complemented by off-chain metadata that can be fetched from off-chain metadata servers.

What are these metadata servers?  Do they already exist & are they described elsewhere?
- if so, can this information be linked here?
- If not, can a rough plan for these servers be put here & included in the Implementation Plan?